Networking/Repository/AbstractRepository.swift

```diff
@@ -249,6 +238,32 @@ private extension AbstractRepository {
         return SignalProducer(error: .requestError(error))
     }
     
+    func tryPollingRequest<T>(
+        method: NetworkingMethod,
+        path: String,
+        tryNumber: Int,
```

Comment:
I would give it default value 0

Reply:
We could but I think there's no use case where you would want to use it without specifying the `tryNumber`, the actual use of the polling is to call the public polling method that calls this one with `tryNumber` = 0 because its initializing the request (this one is private).
It's done recursively to avoid adding state to the repository.

Reply:
Yeah, that's why I thought the default would be a nice thing. Because any other function calls it without the tryNumber and only this function uses it because he knows he's recursive. But it was just an idea, because even if it has a default value, from the outside it could be could with tryNumber = 5 for example.